drivers/mrf24j40/mrf24j40_internal.c

```diff
@@ -87,6 +87,11 @@ void mrf24j40_init(mrf24j40_t *dev)
     mrf24j40_reg_write_short(dev, MRF24J40_REG_BBREG2, MRF25J40_BBREG2_CCAMODE1 );
     mrf24j40_reg_write_short(dev, MRF24J40_REG_CCAEDTH, 0x60);
     mrf24j40_reg_write_short(dev, MRF24J40_REG_BBREG6, MRF24J40_BBREG6_RSSIMODE2 );
+#if MRF24J40_USE_EXT_PA_LNA
+    mrf24j40_reg_write_long(dev, MRF24J40_REG_TESTMODE, (MRF24J40_TESTMODE_TESTMODE2 |
+                                                         MRF24J40_TESTMODE_TESTMODE1 |
+                                                         MRF24J40_TESTMODE_TESTMODE0));
```

Comment:
Adding `MRF24J40_TESTMODE_RSSIWAIT0` here to make 0x0F like in the Microchip driver increased the RSSI on a receiving samr21-xpro from -38 dBm to -34 dBm.

The datasheet just refers to this as the "Optimized value (default)".
Why it switches to `MRF24J40_TESTMODE_RSSIWAIT1` for Clear Channel Assessment is still unknown to me.